sandbox/env/src/lib.rs

```diff
@@ -21,9 +21,15 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 use codec::{Decode, Encode};
-
 use sp_core::RuntimeDebug;
 use sp_std::vec::Vec;
+use sp_wasm_interface::ReturnValue;
+
+#[derive(Clone, Copy, Debug)]
+pub enum Instantiate {
+    Version1,
+    Version2,
```

Comment:
I think it would be a good practice to describe difference between versions

Reply:
Comment would be awesome, agree @ark0f